icuSources/include/_foundation_unicode/utypes.h

```diff
@@ -354,13 +354,13 @@ typedef double UDate;
 #endif
 
 #if defined(U_COMBINED_IMPLEMENTATION)
-#define U_DATA_API     U_EXPORT
-#define U_COMMON_API   U_EXPORT
-#define U_I18N_API     U_EXPORT
-#define U_LAYOUT_API   U_EXPORT
-#define U_LAYOUTEX_API U_EXPORT
-#define U_IO_API       U_EXPORT
-#define U_TOOLUTIL_API U_EXPORT
+#define U_DATA_API     __attribute__((visibility("hidden")))
```

Comment:
Should we leave a comment in the code here explaining why this isn't exported as API?

Reply:
Good point! I shall